doc/python/orca-management.md

```diff
@@ -207,4 +259,4 @@ In addition to the `executable` property, the `plotly.io.orca.config` object can
 
```

Comment:
line 253 should be `- **`default_scale`**: The default image scale factor applied on image export.`

Reply:
Fixed in 8731ecfe3402002d1eac53e74e7d7ce821b2c821